src/NWCClient.ts

```diff
@@ -235,9 +235,10 @@ export class NWCClient {
   lud16: string | undefined;
   walletPubkey: string;
   options: NWCOptions;
-  version: string | undefined;
+  encryptionOrVersion: string | undefined;
 
   static SUPPORTED_VERSIONS = ["0.0", "1.0"];
+  static SUPPORTED_ENCRYPTIONS = ["nip04", "nip44_v2"];
```

Comment:
can you add a TODO to the version code `// TODO: remove after 2026-01-01` or something